ClientCore/ClientCore.csproj

```diff
@@ -50,9 +50,11 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Rampastring.Tools" Version="2.0.4" />
-    <PackageReference Include="Rampastring.XNAUI.$(Engine)" Version="2.3.8" Condition="'!$(Configuration.Contains(Debug))'" />
-    <PackageReference Include="Rampastring.XNAUI.$(Engine).Debug" Version="2.3.8" Condition="'$(Configuration.Contains(Debug))'" />
+    <!-- TODO: update version after this package targets .NET 8. -->
+    <PackageReference Include="Rampastring.Tools" Version="2.0.6-local-397fda5" />
+    <!-- TODO: update version after this package targets .NET 8. -->
```

Comment:
Nit: Do we have a ticket or GitHub task anywhere for this? I generally don't like hanging todos, even ones this obvious.

Reply:
Well, there are nearly thirties TODOs in the codes. Codes with unfinished TODOs are better than say nothing. And some TODOs might remain unfinished forever, because of, for example, a rare usecase.